src/app/utils/string-operations.ts

```diff
@@ -0,0 +1,11 @@
+declare global {
+  interface String {
+    toSentenceCase(): String;
+  }
+}
+
+String.prototype.toSentenceCase = function () {
```

Comment:
This is nice! I didn't know this can be done

Reply:
Does it get pushed to intellisense?

Reply:
It does.